apps/fz_http/lib/fz_http/devices.ex

```diff
@@ -172,19 +169,17 @@ defmodule FzHttp.Devices do
   def as_encoded_config(device), do: Base.encode64(as_config(device))
 
   def as_config(device) do
-    wireguard_port = Application.fetch_env!(:fz_vpn, :wireguard_port)
-    server_public_key = Application.get_env(:fz_vpn, :wireguard_public_key)
+    wireguard_port = get_default_wireguard_port()
+    server_public_key = Gateways.get_gateway!().public_key
```

Comment:
Does it mean that all gateway servers will have the same public and private keys?

Reply:
This is just temporary, when we support multiple gateways will change to `Gateways.get_gateway(device.gateway_id).public_key`